lib/container_orchestrator/object_definition.rb

```diff
@@ -176,9 +178,24 @@ def default_environment
         {:name => "WORKER_HEARTBEAT_FILE",   :value => Rails.root.join("tmp/worker.hb").to_s},
         {:name => "WORKER_HEARTBEAT_METHOD", :value => "file"},
         {:name => "TERRAFORM_RUNNER_URL",    :value => "https://opentofu-runner:6000"},
+        {:name => "SECRET_KEY", :value => opentofu_runner_secret_key},
```

Comment:
Should this ENV var be prefixed with `TERRAFORM_RUNNER_` like the others?

Reply:
Agree, plus this should probably be a secret instead of an ENV var

Reply:
Do we need SECRET_KEY defined on every container or just the opentofu-runner?

Reply:
Agreed, I think this should only be added to the opentofu-runner.  Yes, I think it needs to be a volume mount otherwise it may be flagged by security scans.